resources/scripts/routers/DashboardRouter.tsx

```
@@ -214,4 +223,4 @@ function DashboardRouter() {
     );
 }
 
-export default DashboardRouter;
+export default DashboardRouter;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)